web/db/migrations/20260901120000_remove_blaxel_vm_provider/migration.sql

```
@@ -13,10 +13,10 @@
 -- Blaxel machines BEFORE applying this migration — afterwards nothing in the
 -- control plane can address them.
 
-UPDATE "cloud_vms" SET "provider" = 'e2b' WHERE "provider" = 'blaxel';
-UPDATE "cloud_vm_usage_events" SET "provider" = 'e2b' WHERE "provider" = 'blaxel';
-UPDATE "cloud_vm_bases" SET "active_provider" = 'e2b' WHERE "active_provider" = 'blaxel';
-UPDATE "cloud_vm_base_generations" SET "provider" = 'e2b' WHERE "provider" = 'blaxel';
+UPDATE "cloud_vms" SET "provider" = 'e2b' WHERE "provider"::text = 'blaxel';
```

SQLFluff (4.3.0)

[error] The 'WHERE' keyword should always start a new line.

(LT14)